pkgs/top-level/all-packages.nix

```diff
@@ -27479,6 +27479,14 @@ in
 
   octopus = callPackage ../applications/science/chemistry/octopus { };
 
+  openlp = libsForQt5.callPackage ../applications/misc/openlp { };
+  openlpFull = openlp.override {
```

Comment:
different name attribute is needed as well for nix-env

Reply:
I fixed it.

Reply:
> different name attribute is needed as well for nix-env

...which applies maybe also for a couple of other packages:

`[ "connmanFull" "curlFull" "mjpegtoolsFull" "octaveFull" "pythonFull" "python2Full" "python27Full" "python3Full" "python36Full" "python37Full" "python38Full" "python39Full" "krb5Full" "pulseaudioFull" "samba4Full" "sambaFull" "gitFull" "polybarFull" ]`

says

```nix
{ pkgs ? import ./. {} }:
let
  attrNames = 
    [ "connman" "curl" "mjpegtools" "octave"
      "python" "python2" "python27" "python3" "python36" "python37" "python38" "python39"
      "harfbuzz" "krb5" "pulseaudio" "samba4" "samba" "codeblocks" "git" "polybar"
      "mercurial" # mercurial is the negative test case: its plain and Full names are distinct
    ];
  filtered = builtins.filter (n: pkgs."${n}".name == pkgs."${n}Full".name) attrNames;
in map (n: "${n}Full") filtered
```

For which of those is it unnecessary?

Reply:
(I got the potential packages by `grep Full pkgs/top-level/all-packages.nix`)